src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DAY_POSITION = 0;
+    private static final int NAME = 1;
+    private static final int HOUR = 2;
+    private static final int SALARY_PER_HOUR = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, DATE_FORMATTER);
+        LocalDate toDate = LocalDate.parse(dateTo, DATE_FORMATTER);
+        StringBuilder salaryInfo = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo);
```

Comment:
Consider using StringBuilder for concatenating strings within the loop to improve performance. However, in this case, since it's outside of the loop, it's acceptable to use simple concatenation.